### ISSUE TITLE
install.sh fails after rss feed request

### ISSUE THREAD

**Issue by bverc** [alombarte/raspberry-osmc-automated#28]
Using latest OSMC 2018.03-2

After running the install script, the install crashes out with no information after asking for the RSS feed (regardless of providing or using default).

Perhaps related to #27 when I removed that line it provided more information:

```
install.sh: line 52: syntax error near unexpected token `('
install.sh: line 52: `IP_GUESS=$(ifconfig | awk '/inet addr/{print substr($2,6)}' | grep -v 127.0.0.1)'
```

**Comment by alombarte** [comment]
Hi, I still have the same machine running! But I kept installing the updates and what I see is that the output of the ifconfig has changed. This line `52` should be something like this, can you try it?

`IP_GUESS=$(ifconfig | awk '/inet /{print $2}' | grep -v 127.0.0.1)`

**Comment by bverc** [comment]
Yea thanks, that works. Although it took me a long time to work out why my edits weren't saving. Turns out line 29 redownloads the file over itself. Had to delete that line before it would work.

**Comment by alombarte** [comment]
I am buying a new Raspberry 3 at some point soon and will refresh the project :)

**Comment by alombarte** [comment]
Hi @bverc,
I received my new raspberry 3, downloaded the image and launched the whole script again. My new box is now running. I thought that I would need to do more tweaking but it seems to work as it did 3 years ago. 

The problems I found are:
1) The `IPGUESS` as you reported. I added a `tail` at the end
2) The well known `403: Forbidden in transmission.` -> I think that when the script that modifies the allowed IPs runs the transmission file in `.config/transmission-daemon/settings.json`still doesn't exist. I started and stopped the daemon before doing the replacement. That should work.

If I find more issues while using it "in production" in the following days will push the fixes.

Thanks for reporting!

**Comment by bverc** [comment]
Yea I think you covered the two major things I needed to change.

Interesting about transmission, as mine had added the required ip address mask already when I edited it. However the hostname whitelist was still enabled. I wonder if a question should be added to disable hostname whitelist.

Only other suggestion would be to update the showrss example as the feed url structure seems to have changed (but I assume old feeds still work)
